jwst/datamodels/model_base.py

```diff
@@ -274,6 +273,10 @@ def __repr__(self):
 
         return "".join(buf)
 
+    def __del__(self):
+        """Ensure closure of resources when deleted."""
+        self.close()
```

Comment:
Does this do anything?  I.e. shouldn't `close` already have been called before the `__del__` destructor ever gets invoked?

Reply:
This is to handle the situation of when a `DataModel` is deleted without being closed, usually when the object is garbage collected.  This was prompted by unit tests.

However, as before, there is still some discussion to be had around close vs deletion, as noted in JP-1568